h5c/vic/src/vic-webapp/src/app/vch-view/vch-view.component.ts

```diff
@@ -225,6 +231,50 @@ export class VicVchViewComponent implements OnInit, OnDestroy {
     this.vmViewService.getContainersData({}); // TODO: filter containers for the vch
   }
 
+   /**
+   * Download Certificate for VCH
+   */
+  downloadVchCert(vch: VirtualContainerHost) {
+    this.isDgLoading = true;
+    Observable.combineLatest(
+      this.createWzService.getVicApplianceIp(),
+      this.createWzService.acquireCloneTicket()
+    ).catch(err => {
+      return Observable.throw(err);
+    }).subscribe(([serviceHost, cloneTicket]) => {
+      const vchId = vch.id.split(':')[3];
+      const servicePort = VIC_APPLIANCE_PORT;
+      const targetHost = this.extSessionService.getVcenterServersInfo()[0];
+      const targetHostname = targetHost.name;
+      const targetThumbprint = targetHost.thumbprint;
+      const url =
+        `https://${serviceHost}:${servicePort}/container/target/${targetHostname}/vch/${vchId}/certificate?thumbprint=${targetThumbprint}`;
+
+      const headers  = new Headers({
+        'Content-Type': 'application/json',
+        'X-VMWARE-TICKET': cloneTicket
+      });
+
+      const options  = new RequestOptions({ headers: headers });
```

Comment:
Also not a blocker for 1.3.0.

We should consolidate the code for constructing requests options as well.